__tests__/endpoints/app.test.js

```diff
@@ -173,3 +173,24 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201, and returns the new comment", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: "Lorem fistrum qué dise usteer fistro de la pradera torpedo a wan nostrud al ataquerl.",
+    };
+
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        console.log("body :>> ", body);
+        expect(body.comment.author).toBe("icellusedkars");
+        expect(body.comment.body).toBe(
+          "Lorem fistrum qué dise usteer fistro de la pradera torpedo a wan nostrud al ataquerl."
+        );
```

Comment:
This is a good start but there are a few tests that are missing for this endpoint as well.

What happens when you query for comments on a valid article but one that doesnt have any associated comments? I.e. /api/articles/2/comments 
What should happen when given an article id that is invalid? I.e. /api/articles/banana/comments
What should happen when given an article id that is valid but doesnt exist? I.e. /api/articles/10000/comments
What should happen when the username in the body doesnt exist? 
I.e. 
```
const newComment = {
      username: "icellusedkars",
      body: "Lorem fistrum qué dise usteer fistro de la pradera torpedo a wan nostrud al ataquerl.",
    };
```
What should happen if one of the keys is missing from the given body? 
I.e.
```
const newComment = {
      username: "icellusedkars",
    };
```